mynearwallet/my-near-wallet
# refactor: account inputs

## Description
Refactor account inputs to avoid future potential issues

## Work Done
* Updated from class to function component
* Changed a potential issue with + number and string

## How to Test
* Run the app
* Go to create account 
* Input an account (it should work as before)
* Go to send 
* Input an account to send to (it should work as before)
